src/app/board/board.component.spec.ts

```diff
@@ -24,22 +24,50 @@ describe('BoardComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  describe('board construction', () => {
+  describe('Board is empty', () => {
     it('creates an empty board', () => {
-      expect(component.isBoardEmpty()).toEqual(true);
+      const board = new BoardComponent();
+
+      expect(board.isBoardEmpty()).toEqual(true);
+    });
+
+    it('adds move to an empty board', () => {
+      const board = new BoardComponent();
+
+      board.placeMark('X', 1);
+
+      expect(board.isBoardEmpty()).toEqual(false);
+    });
+  });
+
+  describe('Board is full', () => {
+    it('an empty board that is not full', () => {
+      const board = new BoardComponent();
+
+      expect(board.isBoardFull()).toEqual(false);
+    });
+
+    it('creates a full board', () => {
+      const board = SpecHelper.setupBoard('X', 'X', 'X', 'O', 'X', 'X', 'X', 'X', 'X');
+
+      expect(board.isBoardEmpty()).toEqual(false);
```

Comment:
Perhaps this was a typo? I think you wanted to test whether isBoardFull returns true.
In addition you've already tested the isboardempty method